src/odemis/driver/xt_client.py

```diff
@@ -1209,7 +1206,8 @@ def moveRel(self, shift):
         Parameters
         ----------
         shift: dict(string->float)
-            Relative shift to move the stage to per axes in m. Axes are 'x' and 'y'.
+            Relative shift to move the stage to per axes in m for 'x', 'y', 'z' in rad for 't', 'r'.
+            Axes are 'x', 'y', 'z', 't' and 'r'.
```

Comment:
For moveRel, I think there is a little bit more than nothing to do. In case the rotation supports a full turn, then the absolute value computed should be with modulo 2pi. Otherwise if you're at 5°, and you move -10°, it'll fail.

Reply:
updated